cmd/neofs-cli/modules/container/create.go

```diff
@@ -134,17 +134,21 @@ It will be stored in sidechain when inner ring will accepts it.`,
 			getPrm.SetClient(cli)
 			getPrm.SetContainer(id)
 
-			for i := 0; i < awaitTimeout; i++ {
+			for {
 				time.Sleep(1 * time.Second)
```

Comment:
change it to a channel now (here and in other places)? as i understand it will block sigint for 1 sec